survey/src/test/kotlin/com/surveyor/survey/SurveyApplicationTests.kt

```diff
@@ -0,0 +1,34 @@
+package com.surveyor.survey
+
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.http.MediaType
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import org.springframework.test.web.servlet.ResultActions
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class SurveyApplicationTests @Autowired constructor(val mockMvc: MockMvc) {
+
+    @Test
+    fun testGetSurvey() {
```

Comment:
```suggestion
    fun `should get survey if exists`() {
```
Becomes more understandable If you followed should conventions for testing